### PR TITLE
Feature/booster shot per age group

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard22**

### CODE REVIEW

**Review by VWSCoronaDashboard18 — APPROVED** [review]

File: packages/app/src/domain/vaccine/vaccine-coverage-per-age-group/components/narrow-coverage-table.tsx [hunk 1]
```diff
@@ -4,20 +4,28 @@ import { useIntl } from '~/intl';
 import { formatAgeGroupString } from '~/utils/format-age-group-string';
 import { formatBirthyearRangeString } from '~/utils/format-birthyear-range-string';
 import { useVaccineCoveragePercentageFormatter } from '../../logic/use-vaccine-coverage-percentage-formatter';
+import { COLOR_FULLY_VACCINATED, COLOR_HAS_ONE_SHOT } from '../../common';
```
Comment (VWSCoronaDashboard19) [inline packages/app/src/domain/vaccine/vaccine-coverage-per-age-group/components/narrow-coverage-table.tsx:7]: You can use ~/domain/vaccine here for the import.

File: packages/app/src/domain/vaccine/vaccine-coverage-per-age-group/components/narrow-coverage-table.tsx [hunk 2]
```diff
@@ -4,20 +4,28 @@ import { useIntl } from '~/intl';
...
+import { Bar } from '../../components/bar';
```
Comment (VWSCoronaDashboard19) [inline packages/app/src/domain/vaccine/vaccine-coverage-per-age-group/components/narrow-coverage-table.tsx:8]: Same here `~/components/bar`
A bit more readable

File: packages/app/src/domain/vaccine/booster-shot-coverage-per-age-group/components/wide-coverage-table.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+import css from '@styled-system/css';
+import styled from 'styled-components';
+import { Box } from '~/components/base';
+import { InlineText } from '~/components/typography';
+import { useIntl } from '~/intl';
+import { SiteText } from '~/locale';
+import { asResponsiveArray } from '~/style/utils';
+import { formatAgeGroupString } from '~/utils/format-age-group-string';
+import { formatBirthyearRangeString } from '~/utils/format-birthyear-range-string';
+import { COLOR_FULLY_BOOSTERED } from '../../common';
```
Comment (VWSCoronaDashboard19) [inline packages/app/src/domain/vaccine/booster-shot-coverage-per-age-group/components/wide-coverage-table.tsx:11]: Please use `~` for the imports as defined in tsconfig. `~/domain/vaccine/common`

File: packages/app/src/domain/vaccine/vaccine-coverage-per-age-group/components/wide-coverage-table.tsx [hunk 4]
```diff
@@ -12,10 +12,10 @@ import { asResponsiveArray } from '~/style/utils';
 import { formatAgeGroupString } from '~/utils/format-age-group-string';
 import { formatBirthyearRangeString } from '~/utils/format-birthyear-range-string';
 import { useVaccineCoveragePercentageFormatter } from '../../logic/use-vaccine-coverage-percentage-formatter';
-import { COLOR_FULLY_VACCINATED, COLOR_HAS_ONE_SHOT } from '../common';
-import { AgeGroup } from './age-group';
-import { Bar } from './bar';
-import { WidePercentage } from './wide-percentage';
+import { COLOR_FULLY_VACCINATED, COLOR_HAS_ONE_SHOT } from '../../common';
+import { Bar } from '../../components/bar';
```
Comment (VWSCoronaDashboard19) [inline packages/app/src/domain/vaccine/vaccine-coverage-per-age-group/components/wide-coverage-table.tsx:16]: Here also the same

**Review by VWSCoronaDashboard19 — CHANGES_REQUESTED** [review]
Small improvements on the import side, for better readability. For the rest code looks good.

**Review by VWSCoronaDashboard19 — APPROVED** [review]

**Pull request merged**